modules/configuring-egress-proxy.adoc

```diff
@@ -6,7 +6,7 @@
 [id="nw-configuring-cluster-wide-proxy_{context}"]
 = Configuring the AWS Load Balancer Operator to trust the certificate authority of the cluster-wide proxy
 
-. Create the config map to contain the certificate authority (CA) bundle in the `aws-load-balancer-operator` namespace by running the following command:
+. Create the config map to contain the certificate authority (CA) bundle in the `aws-load-balancer-operator` namespace and inject a CA bundle that is trusted by {product-title} into a configmap by running the following command:
```

Comment:
@xenolinux, we're calling it _config map_: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#config-map